<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Widget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
    <item>
     <widget class="QGroupBox" name="InformationServer">
      <property name="title">
       <string>Информация о сервер</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,1">
       <item>
        <layout class="QHBoxLayout" name="NameInformationServerLayout" stretch="1,2">
         <item>
          <widget class="QLabel" name="NameInformationServerLabel">
           <property name="text">
            <string>Имя сервера:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameServerInfo">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="AdressInformationServerLayout" stretch="1,2">
         <item>
          <widget class="QLabel" name="AdressInformationServerLabel">
           <property name="text">
            <string>Адрес сервера:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="adressServerInfo">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnConnectToServer">
         <property name="text">
          <string>Подключится</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="ChatLayout">
      <property name="title">
       <string>Чат</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,5,1">
       <item>
        <widget class="QGroupBox" name="UserSettingsLayout">
         <property name="title">
          <string>Пользователь</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
          <item>
           <layout class="QHBoxLayout" name="NickNameLayout" stretch="1,4">
            <item>
             <widget class="QLabel" name="NickNameLabel">
              <property name="text">
               <string>Никнейм:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nick"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="chat">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ChatMessageLayout" stretch="4,1">
         <item>
          <widget class="QLineEdit" name="message"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendMessage">
           <property name="text">
            <string>Отправить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
